packages/fast-components-react-msft/src/call-to-action/call-to-action.tsx

```diff
@@ -89,6 +85,17 @@ class CallToAction extends Foundation<
 
         return super.generateClassNames(classNames);
     }
+
+    private renderAfterContent(): any {
+        return (): React.ReactNode => {
+            return (
+                <div
```

Comment:
nit: inline within button should be a span.

Reply:
@chrisdholt  Hm this would be a shape change... I guess it doesn't really matter because it is internal?

Reply:
It's internal and not exposed by any part of the API, so this isn't breaking whatsoever in my opinion.